src/Menu/Menu.scss

```diff
@@ -12,27 +10,27 @@ $sideTagColor: #f7c244;
 }
 
 .food-item-card {
-    border: solid 2px #898c8e;
+    border: solid 2px $borderColor;
     text-align: left;
     box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.1), 0 6px 10px 0 rgba(0, 0, 0, 0.1);
 
     h2 {
         font-size: 22px;
         font-weight: bold;
     }
-    h3 {
+    .calories-tag, .type-tag, .side-tag {
```

Comment:
i think it would be more future proof instead to use the selector `.tag-container p`, that way if there were any different tags in future the styling would apply to them as well

Reply:
Done!